config.cson

```diff
@@ -2227,7 +2227,7 @@ fileIcons:
 		icon: "elixir"
 		match: [
 			[".ex", "dark-purple", "elixir"]
-			[/\.(exs|l?eex)$/i, "medium-purple"]
+			[/\.(exs|(h|l)?eex)$/i, "medium-purple"]
```

Comment:
**Tiny nitpick:** this can be simplified with a character set, as each of the potential matches is a single character long:
```suggestion
			[/\.(exs|[hl]?eex)$/i, "medium-purple"]
```